docs/hub/security-two-factor-authentication.md

```diff
@@ -0,0 +1,99 @@
+
+## Recovery codes
+
+Right after you've successfully activated 2FA with a one-time password, you're requested to download a collection of generated recovery codes. If you ever lose access to your one-time password authenticator, you can utilize one of these recovery codes to log in to your account.
+
+- Each code can be used only **once** to sign in to your account
+- You should copy and print the codes, or use Download codes to download them for storage in a safe place. If you choose to download them, the file is called **huggingface-recovery-codes.txt**
+
+<!-- blurred screenshot -->
+
+If you lose the recovery codes, or want to generate new ones, you can use the Authentication settings page.
+
+## Regenerate two-factor authentication recovery codes
+
+To regenerate 2FA recovery codes, you need access to a desktop browser:
+1. Access your settings
+2. Select Authentication
+3. If you’ve already configured 2FA, select Recovery Code
+4. Click on Regenerate recovery codes
+
+<Tip warning={true}>
+If you regenerate 2FA recovery codes, save them. You can’t use any previously created 2FA codes.
+</Tip>
+
+## Sign in with two-factor authentication enabled
+
+When you sign in with 2FA enabled, the process is only slightly different than the standard sign-in procedure. After entering your username and password, you'll encounter an additional prompt, depending on the type of 2FA you've set up. When prompted, provide the pin from your one-time password authenticator's app or a recovery code to complete the sign-in process.
+
+<!-- screenshot -->
+
+## Disable two-factor authentication
+
+To disable 2FA:
+1. Access your User settings
+2. Select Authentication
+3. Click on Remove.
+
+This clears all your 2FA registrations.
+
+## Recovery options
+
+If you don’t have access to your code generation device, you can recover access to your account:
+
+- Use a saved recovery code, if you saved them when you enabled two-factor authentication
+- Requesting help with two-factor authentication
+
+### Use a recovery code
+
+To use a recovery code:
+1. Enter your username or email, and password, on the Hub sign-in page
+2. When prompted for a two-factor code, click on "Lost access to your two-factor authentication app? Use a recovery code"
+3. Enter one of your recovery code
+
+After you use a recovery code, you cannot re-use it. You can still use the other recovery codes you saved.
+
+### Requesting help with two-factor authentication
+
+In case you've forgotten your password and lost access to your two-factor authentication credentials, you can reach out to support (website@huggingface.co) to regain access to your account. You'll be required to verify your identity using a recovery authentication factor, such as an SSH key.
+
+To complete your recovery request, you'll need to confirm an alternative authentication factor. Choose a recovery verification method:
+- If you've previously established an SSH key on this account, provide your public SSH key
```

Comment:
note that there was an open issue (by @severo i think) to expose any user's SSH public keys 

So in that case anyone would have this

Reply:
https://github.com/huggingface/moon-landing/issues/8160

Reply:
I would just keep the very first line of this section for now I think :)